testing-examples/chatbot-simulation/chatbot-simulation.ipynb

```diff
@@ -304,9 +304,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from langchain.smith import RunEvalConfig\n",
     "from langchain_core.prompts import ChatPromptTemplate, MessagesPlaceholder\n",
-    "from langchain_core.pydantic_v1 import BaseModel, Field\n",
+    "from pydantic import BaseModel, Field\n",
```

Comment:
Switched from `langchain_core.pydantic_v1` to `pydantic` directly to resolve warnings:
- Pydantic BaseModel V1 schema compatibility warning

For more details: https://python.langchain.com/docs/versions/v0_3/#1-do-not-use-the-langchain_corepydantic_v1-namespace